build/source/driver/multi_driver.f90

```diff
@@ -247,77 +357,123 @@ real(dp),parameter        :: safety = 0.01_dp               ! safety factor to e
  case default; call handle_err(30,'unable to identify vegetation category')
 end select
 
+! allocate space for default model parameters
+if(allocated(dparStruct%hru))then
+ call handle_err(20,'dparStruct is unexpectedly allocated')
+else
+ ! allocate the spatial dimension
+ allocate(dparStruct%hru(nHRU),stat=err)
+ if(err/=0) call handle_err(20,'problem allocating dparStruct')
+endif
+
+! set default model parameters
+do iHRU=1,nHRU
+ ! allocate the variable dimension for a given HRU
+ call allocLocal(mpar_meta,dparStruct%hru(iHRU),err=err,message=message)
+ call handle_err(err,message)
```

Comment:
Maybe use the convention: If the original call is a one-liner then the call to `handle_err` is on the same line. Otherwise it comes on its own line.